lib/navigation/model/route_paths.dart

```diff
@@ -0,0 +1,347 @@
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
```

Comment:
When you have the final design for the routing, please document it in CONTRIBUTING.md. It's important to have some instructions about how to properly create a new page/route (not in this PR).

(it may also be worth introducing some automated checks, e.g. if someone uses Navigator instead of AppNavigator to refer them to the routing documentation; and on that note -- what happens if e.g. someone does use AppNavigator but didn't implement a route in this file? we should try to make the error message informative so people don't get stuck on this)

Reply:
I created an issue to not forget about this. I am planning to merge this PR in a couple of minutes